app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(
+        self,
+        name: str,
+        age: int,
+    ) -> None:
+        self.name = name
+        self.age = age
+        new_instance = {name: self}
+        self.people.update(new_instance)
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_objects = []
+    for person in people:
+        new_person = Person(person["name"], person["age"])
+        for key, value in person.items():
+            if key == "wife" and value is not None:
+                setattr(new_person, key, value)
+            elif key == "husband" and value is not None:
+                setattr(new_person, key, value)
+        new_objects.append(new_person)
```

Comment:
create list of Person using list comprehension

Reply:
sorry, I have no ideas how to do this, could you please explain here?